icon.css

```diff
@@ -0,0 +1,23 @@
+.icon.blue{
+    color: background-grey;
```

Comment:
For these to take effect, you just need to wrap them in `var()` and add `--` before them.

```
color: var(--background-grey);
```

You can see all of these variables defined in `src/styles/variables.css`.